lib/streamlit/elements/dataframe_selector.py

```diff
@@ -296,8 +296,8 @@ def altair_chart(self, altair_chart, use_container_width=False):
         https://altair-viz.github.io/gallery/.
 
         .. output::
-           https://static.streamlit.io/0.25.0-2JkNY/index.html?id=8jmmXR8iKoZGV4kXaKGYV5
-           height: 200px
+           https://share.streamlit.io/streamlit/docs/main/python/api-examples-source/charts.vega_lite_chart.py
```

Comment:
Is this supposed to be vega_lite_chart or is there an Altair specific one? Okay if there isn't, just want to make sure it's not a copy/paste error

Reply:
This is intentional. Visually, there's no difference in the output so i use the same app.